lib/facter/gluster.rb

```diff
@@ -61,28 +61,28 @@
   if peer_count > 0
     Facter.add(:gluster_peer_list) do
       setcode do
-        peer_list
+        peer_list.sort
       end
     end
 
     unless volume_bricks.empty?
       Facter.add(:gluster_volume_list) do
         setcode do
-          volume_bricks.keys
+          volume_bricks.keys.sort
         end
       end
       volume_bricks.each do |vol, bricks|
         Facter.add("gluster_volume_#{vol}_bricks".to_sym) do
           setcode do
-            bricks
+            bricks.sort
```

Comment:
You are right. After a short look to the documentation I'd say the brick order is important for stripped and distributed volumes. So it looks like #121 isn't a bug in this module but in the puppet configuration of the author of #121.